### PR TITLE
Add Manila service deployment to install_yaml

### PR DESCRIPTION
**Pull request by fmount**
This patch adds the required workflow to deploy manila and the related services using install_yaml project.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -173,6 +173,13 @@ DATAPLANE_BRANCH     ?= main
 # Ceph
 CEPH_IMG       ?= quay.io/ceph/demo:latest
 
+# Manila
+MANILA_IMG       ?= quay.io/openstack-k8s-operators/manila-operator-index:latest
```
Comment (fmount) [inline Makefile:177]: This is still not present in the `quay.io/openstack-k8s-operators` namespace

File: Makefile [hunk 2]
```diff
@@ -1028,6 +1035,7 @@ ceph: namespace ## deploy the Ceph Pod
 	bash scripts/gen-ceph-kustomize.sh "build"
 	oc kustomize ${DEPLOY_DIR} | oc apply -f -
 	bash scripts/gen-ceph-kustomize.sh "isready"
+	bash scripts/gen-ceph-kustomize.sh "cephfs"
```
Comment (fmount) [inline Makefile:1038]: @fultonj a new function creating a volume (called "cephfs") for manila here

Comment (fultonj) [reply Makefile:1038]: The scripting changes look reasonable to me.

Do you want to update a README with info on how to use the manila operator?

File: Makefile (hunk 1 above)
Comment (fultonj) [reply Makefile:177]: do you want to wait until that change is present in the namespace before merging?

Comment (fmount) [reply Makefile:177]: @dprince should we `make catalog` and `make catalog-push` to `quay.io/openstack-k8s-operators` ? I think I don't have enough permissions to push it, but apart from that, it should be ready to go.

Comment (fmount) [reply Makefile:177]: Fixed by [1] and now we have all we need.

https://quay.io/repository/openstack-k8s-operators/manila-operator-bundle?tab=tags&tag=latest
https://quay.io/repository/openstack-k8s-operators/manila-operator-index?tab=tags&tag=latest
https://quay.io/repository/openstack-k8s-operators/manila-operator?tab=tags&tag=latest

[1] https://github.com/openstack-k8s-operators/manila-operator/pull/37

**Comment by fmount** [comment]
Tested and works as expected.

**Review by dprince — APPROVED** [review]

**Pull request merged**